File: Gico System/dev/Gico.HtmlApps/SystemAdmin/src/app/models/variation-theme-model.ts
import { ProductAttributeModel } from "./product-attr-model";



export class VariationThemeModel{
    variationThemeId:number;
    variationThemeName:string;
    status:number;
}

export class VariationThemeChooseModel{
    name:string;
    id:number;
    attributes:ProductAttributeModel[];
}

export class Category_VariationTheme_AddRequest{
    variationThemeId:number[];
    categoryId:string 
}

export class Category_VariationThemeMapping{
    variationThemeId:number;
    categoryId : string;
    variationThemeName:string
}

File: Gico System/dev/Gico.SystemModels/Models/VariationThemeModel.cs
using System;
using Gico.Config;
using Gico.Models.Models;
namespace Gico.SystemModels.Models
{
    public class VariationThemeModel : BaseModel
    {
        public int VariationThemeId { get; set; }
        public string VariationThemeName { get; set; }
        public new EnumDefine.VariationThemeStatus Status { get; set; }
    }

    public class VariationTheme_AttributeModel : BaseModel
    {
        public int VariationThemeId { get; set; }
        public string VariationThemeName { get; set; }

        public ProductAttributeModel[] Attributes { get; set; }

    }
    public class Category_VariationTheme_MappingModel : BaseModel
    {
        public int[] VariationThemeId { get; set; }
        public string CategoryId { get; set; }

    }

    public class CategoryVariationThemeMappingModel : BaseModel
    {
        public int VariationThemeId { get; set; }
        public string CategoryId { get; set; }

        public string VariationThemeName { get; set; }

    }

}
